<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>iriverPLA v0.1.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/icon.ico</normaloff>:/icons/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="rootLabel">
        <property name="text">
         <string>Root:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="rootPathEntry"/>
        </item>
        <item>
         <widget class="QPushButton" name="rootPathSelButton">
          <property name="text">
           <string>Select</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="musicTreeLabel">
            <property name="text">
             <string>Music:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTreeView" name="musicTreeView"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="addButton">
            <property name="text">
             <string>&gt;</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="delButton">
            <property name="text">
             <string>&lt;</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/del.png</normaloff>:/icons/del.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="musicListLabel">
          <property name="text">
           <string>Music:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="musicListView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="playlistLabel">
        <property name="text">
         <string>Playlist:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="playlistEntry"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="saveListButton">
        <property name="text">
         <string>Save List</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearListButton">
        <property name="text">
         <string>Clear List</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/clear.png</normaloff>:/icons/clear.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="createButton">
        <property name="text">
         <string>Create</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/create.png</normaloff>:/icons/create.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="quitButton">
        <property name="text">
         <string>Quit</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <zorder></zorder>
   <zorder>splitter</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>645</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_list"/>
    <addaction name="actionSave_list"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionCreate_PLA"/>
    <addaction name="actionClear_list"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionIriverPLA_Help"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionAbout_iriverPLA"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_list"/>
   <addaction name="actionSave_list"/>
   <addaction name="separator"/>
   <addaction name="actionClear_list"/>
   <addaction name="separator"/>
   <addaction name="actionCreate_PLA"/>
   <addaction name="separator"/>
   <addaction name="actionPreferences"/>
   <addaction name="separator"/>
   <addaction name="actionIriverPLA_Help"/>
   <addaction name="actionAbout_iriverPLA"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_list">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open list</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/pref.png</normaloff>:/icons/pref.png</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionIriverPLA_Help">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/help.png</normaloff>:/icons/help.png</iconset>
   </property>
   <property name="text">
    <string>iriverPLA Help</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/about.png</normaloff>:/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAbout_iriverPLA">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/about.png</normaloff>:/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>About iriverPLA</string>
   </property>
  </action>
  <action name="actionSave_list">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save list</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCreate_PLA">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/create.png</normaloff>:/icons/create.png</iconset>
   </property>
   <property name="text">
    <string>Create PLA</string>
   </property>
  </action>
  <action name="actionClear_list">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/clear.png</normaloff>:/icons/clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear list</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
